src/test/store/profile-view.test.js

```diff
@@ -116,83 +123,613 @@ describe('call node paths on implementation filter change', function() {
   });
 });
 
-describe('expand all call node descendants', function() {
-  const {
-    profile,
-    funcNamesPerThread: [funcNames],
-  } = getProfileFromTextSamples(`
-    A A A
-    B B E
-    C D
-  `);
-  const threadIndex = 0;
-  const A = funcNames.indexOf('A');
-  const B = funcNames.indexOf('B');
-  const C = funcNames.indexOf('C');
-  const D = funcNames.indexOf('D');
-  const E = funcNames.indexOf('E');
+/**
+ * The following tests naively runs through a dispatch and selector to provide coverage
+ * over the Redux store to ensure that it behaves correctly.
+ */
+describe('actions/ProfileView', function() {
+  describe('changeSelectedCallNode', function() {
+    const { profile } = getProfileFromTextSamples(`
+      A
+      B
+      C
+    `);
+    const { dispatch, getState } = storeWithProfile(profile);
```

Comment:
I missed this: I think this line should be inside the `it` block.

This doesn't make a difference **now** because there's only one `it` in this `describe` block like in most of this file, but this will make a difference if we add a new `it` case inside the `describe` block because then the 2 cases would be interdependent because they'd share the same store.

So I'd rather go the path of least surprise and move this line now to inside the `it` block.

And the same everywhere in this file of course.

Reply:
I'm making this change, I also filed #846 to have some additional discussion around testing.